src/main/java/appeng/client/gui/implementations/GuiAmount.java

```diff
@@ -0,0 +1,159 @@
+package appeng.client.gui.implementations;
+
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.gui.GuiTextField;
+import net.minecraft.inventory.Container;
+import net.minecraft.item.ItemStack;
+
+import org.lwjgl.input.Keyboard;
+
+import appeng.client.gui.AEBaseGui;
+import appeng.client.gui.widgets.GuiTabButton;
+import appeng.container.AEBaseContainer;
+import appeng.core.AEConfig;
+import appeng.core.localization.GuiColors;
+import appeng.core.localization.GuiText;
+import appeng.core.sync.GuiBridge;
+import appeng.core.sync.network.NetworkHandler;
+import appeng.core.sync.packets.PacketSwitchGuis;
+import appeng.helpers.Reflected;
+import appeng.util.calculators.ArithHelper;
+import appeng.util.calculators.Calculator;
+
+public abstract class GuiAmount extends AEBaseGui {
+
+    protected GuiTextField amountTextField;
+    protected GuiTabButton originalGuiBtn;
+
+    protected GuiButton nextBtn;
+
+    protected GuiButton plus1;
+    protected GuiButton plus10;
+    protected GuiButton plus100;
+    protected GuiButton plus1000;
+    protected GuiButton minus1;
+    protected GuiButton minus10;
+    protected GuiButton minus100;
+    protected GuiButton minus1000;
+
+    protected GuiBridge originalGui;
+    protected ItemStack myIcon;
+
+    @Reflected
+    public GuiAmount(final Container container) {
+        super(container);
+    }
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public void initGui() {
+        super.initGui();
+
+        final int a = AEConfig.instance.craftItemsByStackAmounts(0);
+        final int b = AEConfig.instance.craftItemsByStackAmounts(1);
+        final int c = AEConfig.instance.craftItemsByStackAmounts(2);
+        final int d = AEConfig.instance.craftItemsByStackAmounts(3);
```

Comment:
Please, make sure that correct config is used and make this base class compatible with https://github.com/GTNewHorizons/AE2FluidCraft-Rework/pull/192 (to allow changing config).